bin/uno

```diff
@@ -93,6 +93,14 @@ ashell)
   check_dirs
   "$ACCUMULO_HOME"/bin/accumulo shell -u "$ACCUMULO_USER" -p "$ACCUMULO_PASSWORD" "${@:2}"
 	;;
+astart)
+  check_dirs
```

Comment:
Would it make sense to have a sanity check to see if `setup` was run?  Does `check_dirs` do this?